hs_collection_resource/views.py

```diff
@@ -0,0 +1,106 @@
+import json
+import logging
+
+from django.http import HttpResponseRedirect, HttpResponseNotFound, HttpResponse, JsonResponse
+
+from hs_core import hydroshare
+from hs_core.views import _set_resource_sharing_status
```

Comment:
If you want to use this function (_set_resource_sharing_status) from hs_core, then change it to a public function (set_resource_sharing_status) in hs_core.